Adding 5 new requirements based on #189 disscussion. the changes are … (#274)
OWASP/owasp-masvs: Document/0x06-V1-Architecture_design_and_threat_modelling_requireme.md

| **1.8** | MSTG‑ARCH‑8 | There is an explicit policy for how cryptographic keys (if any) are managed, and the lifecycle of cryptographic keys is enforced. Ideally, follow a key management standard such as NIST SP 800-57. |   | ✓ |
| **1.9** | MSTG‑ARCH‑9 | A mechanism for enforcing updates of the mobile app exists. |   | ✓ |
| **1.10** | MSTG‑ARCH‑10 | Security is addressed within all parts of the software development lifecycle. |   | ✓ |

> | **1.11** | MSTG‑ARCH‑11 | After each security updated, oudated application version should no logger be available in App-stores. and updates are forced on user's device. |   | ✓ |

Reviewer: ```suggestion
| **1.11** | MSTG‑ARCH‑11 | When an app received a security update, outdated versions should no longer be available, and updates should be enforced on the user's device. |   | ✓ |
```

Reviewer 2: I would suggest to remove this requirement. 

1. For the first part

> When an app received a security update, outdated versions should no longer be available

Is it actually possible to download old versions via the Play Store or App Store? Isn't it anyway always the latest version? 
And if it's not via the official channel, of course you can download IPA/APK and sideload it, but how do you want to enforce that outdated versions are not available? They are mirrored via many sites.

2. The 2nd part 
> and updates should be enforced on the user's device. 
Is redundant to requirement 1.9.

Reviewer: Let's remove this indeed

Reviewer: (This means we will "ignore" enterprise app stores for now)

Author: Let's keep the excel aside for a while. merge this changes first.
Then i will pus the excels with the MSTG-ID included and the V1 covered
Then let's have an other PR to add the new requirements this way handling will be easy and we won't have to do too much changes every now and then.
